aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -369,18 +369,16 @@ static std::vector<Dimname> compute_matmul_outnames(
   return result;
 }
 
-void propagate_names_for_addmv(
-    Tensor& result,
+std::vector<Dimname> propagate_names_for_addmv(
```

Comment:
consider using a dimvector here

Reply:
That would require changing all functions using `unify_from_right` (because it returns std::vector<Dimname>), better done as a separate PR